### ISSUE TITLE
Parsing fails with NullPointerException

### ISSUE THREAD

**Issue by arichiardi** [kkinnear/zprint#199]
Hi Kim, I have a very strange problem here with:

```
zprint <<< '(is (= #::sut{:groups #{"FOO_ADMIN" "FOO_USER"}}
         (sut/map->user {:groups ["FOO_USER" "FOO_ADMIN"]})))'
(is (= #::sut{:groups #{"FOO_ADMIN" "FOO_USER"}}
         (sut/map->user {:groups ["FOO_USER" "FOO_ADMIN"]})))
Failed to zprint: java.lang.NullPointerException
```

Now if I remove the `#::sut` everything is good:

```
zprint <<< '(is (= {:groups #{"FOO_ADMIN" "FOO_USER"}}
         (sut/map->user {:groups ["FOO_USER" "FOO_ADMIN"]})))'
```

But:

```
zprint <<< '#::sut{:groups #{"ADMIN" "USER"}}'
#::sut{:groups #{"ADMIN" "USER"}}
```

Therefore I am not sure it is the namespaced map syntax that's breaking there.

I have a workaround - not to use that syntax solves it in some place but I thought I should report it :smile: 

Thanks as usual for your hard work on `zprint`!

**Comment by kkinnear** [comment]
Thanks for noticing this!  Namespaced map syntax does not play well with the currently shipping version of `rewrite-clj`, but the new version (`V1`) by @lread has fixed this, so it will work in the next zprint release: `1.2.0`. Assuming that I work up the ... to release zprint with an alpha version of `rewrite-clj V1`.  But it runs all 1200+ zprint tests, so why not?

**Comment by lread** [comment]
Go for it! 🙂 

I don't expect issues to be caused by rewrite-clj v1, but if we find some, I'll be happy address them.

**Comment by kkinnear** [comment]
This is now fixed by `1.2.0`, which includes `rewrite-clj` V1!
